lib/bindings/python/src/dynamo/llm/vllm_integration/connector/pd_connector.py

```
@@ -0,0 +1,109 @@
+                f"got {type(self._connectors[1]).__name__}"
+            )
+
+    # ==============================
+    # Worker-side methods
+    # ==============================
+
+    def bind_connector_metadata(self, connector_metadata: PdConnectorMetadata) -> None:
+        assert isinstance(connector_metadata, PdConnectorMetadata)
+        if connector_metadata.extra_async_saves:
+            self._extra_async_saves.update(connector_metadata.extra_async_saves)
+        for c, cm in zip(self._connectors, connector_metadata.metadata):
```

Ruff (0.13.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)